index.html

```diff
@@ -0,0 +1,206 @@
+<html>
+    <head>
+        <title>YouTube</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body> 
+
+<nav class="flex-div">
+    <div class="nav-left flex-div">    
+    <img src="images/menu.png" class="menu-icon">
+    <img src="images/logo.png" class="logo">
+        </div>
+        <div class="nav-middle flex-div">
+            <div class="search-box flex-div">
+
+                <input type="search" placeholder="search">
+                <img src="images/search.png" class="search-icon">
+            </div>
+            <img src="images/voice-search.png" class="mic-icon">
+        </div>
+        <div class="nav-right flex-div">
+            <img src="images/upload.png">
+            <img src="images/more.png">
+            <img src="images/notification.png">
+            <img src="images/Jack.png" class="user-icon">
```

Comment:
- [ ] Well done catching these missing `alt`s, however, this comment could be rephrased and framed in a more polite and friendly tone as described in the project [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) 🔽 
> Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it?
- [ ] This comment can also be improved by explaining to the student why this change is important as advised in this project's [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) 🔽 
> Remember to explain why you request a change - it might be obvious for you but not for someone else.